podcasts/AppDelegate.swift

```diff
@@ -35,6 +36,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         configureFirebase()
         TraceManager.shared.setup(handler: traceHandler)
 
+        // Facebook AEM Analytics
+        FacebookCore.Settings.shared.appID = ApiCredentials.instagramAppID
```

Comment:
Let's move the appID settings into the `AppsFlyerAdapter`  `appsFlyerSetup`. Also add the token and app name too.